app/controllers/users/sessions_controller.rb

```diff
@@ -3,4 +3,9 @@
 class Users::SessionsController < Devise::SessionsController
   include Accessible
   skip_before_action :check_user, only: :destroy
+  after_action :after_login, only: :create
+
+  def after_login
+    LoginActivity.create!(user_id: current_user.id, email: current_user.email, current_sign_in_ip: current_user.current_sign_in_ip, current_sign_in_at: Time.now, user_type: current_user.type)
```

Comment:
maybe we should use https://github.com/ankane/authtrail ? What do you think?

Reply:
Had looked into this option but thought as we need to just log history details so don't want to add another gem dependency but can update code if that looks better option in place of custom code?

Reply:
@compwron Have re-implemented functionality using Authtrail gem and raised new PR #5706 for it. Can you please review it whenever possible.. I will close this PR after approval.